### ISSUE TITLE
add_feature, value_exact = FALSE, missing?

### ISSUE THREAD

**Issue by sytpp** [ropensci/osmdata#73]
I had some trouble getting my head around the key_exact parameter in the add_feature function. If I'd like to add various street types, say everything with the key='highway' and value = anything ending in "-ary":

`q <- add_feature(q, key="highway", value="ary", key_exact = TRUE)
`
Using this query my resulting OSM object is empty.

I have to overwrite the q$feature element explicitly to get the output I want.

`q$features[1] <- "[\"highway\"~\"ary\"]"`
  
Is there a different way to achieve basically value_exact=FALSE ?

**Comment by mpadge** [comment]
There's also a `value_exact` boolean parameter which does that:
```
> opq ('somewhere') %>%
        add_feature(key = 'highway', value = "ary", value_exact = FALSE)
$features
[1] " [\"highway\"~\"ary\"]"
```
Is that what you mean?

**Comment by mpadge** [comment]
Oh, but this was only implemented in 0.0.3 (i think), so maybe just make sure you've got latest version installed (0.0.4)?

**Comment by sytpp** [comment]
Oh, right I had the wrong version - thanks a lot!
